### ISSUE TITLE
[Question]: 用的uie-m-base进行训练  max_seq_length 512报太小 1024又爆显存，用A100  40GB显存训练的 

### ISSUE THREAD

**Issue by hurui79** [PaddlePaddle/PaddleNLP#8749]
### 请提出你的问题

os.system('python finetune.py  \
            --device "gpu" \
            --logging_steps 10 \
            --save_steps 1000 \
            --eval_steps 100 \
            --seed 42 \
            --model_name_or_path "./work/uie-m-base" \
            --output_dir ./work/model_best \
            --train_path ./data/train.txt \
            --dev_path ./data/dev.txt \
            --max_seq_length 512 \
            --per_device_eval_batch_size 32 \
            --per_device_train_batch_size  32 \
            --num_train_epochs 20 \
            --learning_rate 1e-5 \
            --export_model_dir ./work/model_best \
            --overwrite_output_dir true \
            --disable_tqdm True \
            --metric_for_best_model eval_f1 \
            --load_best_model_at_end  True \
            --multilingual True \
            --do_train \
            --do_eval \
            --do_export \
            --save_total_limit 1')
pprint("训练完成")

![image](https://github.com/PaddlePaddle/PaddleNLP/assets/16483840/8563e38d-e6f5-469f-bf04-f96ebd30f0f0)

这个是改成1024之后的
![image](https://github.com/PaddlePaddle/PaddleNLP/assets/16483840/ade485e3-f3cd-4b23-b7eb-4acc32ff3d4b)

pip list
![image](https://github.com/PaddlePaddle/PaddleNLP/assets/16483840/5069962a-3f54-4d6f-b002-224a7a28378b)

**Comment by wawltor** [comment]
请提供下AIStudio的任务链接

**Comment by hurui79** [comment]
> 请提供下AIStudio的任务链接

https://aistudio.baidu.com/projectdetail/8151763
我改了一下代码，算是能跑起来了
![a64f2818cfefd599c65bdc98b54c350](https://github.com/user-attachments/assets/8716e9a4-9882-43da-884b-ddb6e6676380)
